src/Moryx.Resources.Management/Resources/ResourceProxy.cs

```diff
@@ -17,6 +19,12 @@ internal abstract class ResourceProxy : IResource
         /// </summary>
         private IResourceTypeController _typeController;
 
+        public event EventHandler<ICapabilities> CapabilitiesChanged
+        {
+            add { Target.CapabilitiesChanged += value; }
```

Comment:
Generally a good idea, but you can not forward the registration as it would pass the non-proxy object as sender to the listener.